tests/Agent/MultiverseTesting/ConsoleScanner/config.yml

```diff
@@ -100,12 +100,12 @@
   #      - { package-name: ServiceStack.Redis, versions: [4.0.40, 4.5.14, 5.10.4] }
   #   local-assemblies:
 
-  # - name: stackExchangeRedis
-  #   xml-file: ${{ MVS_XML_PATH }}/StackExchangeRedis/Instrumentation.xml
-  #   nuget-packages:
-  #      - { package-name: StackExchange.Redis, versions: [1.0.488, 1.1.608, 1.2.6, 2.0.601, 2.2.4] }
-  #      - { package-name: StackExchange.Redis.StrongName, versions: [1.0.488, 1.1.608, 1.2.6] }
-  #   local-assemblies:
+  - name: stackExchangeRedis
+    xml-file: ${{ MVS_XML_PATH }}/StackExchangeRedis/Instrumentation.xml
+    nuget-packages:
+       - { package-name: StackExchange.Redis, versions: [1.0.488, 1.1.608, 1.2.6, 2.0.601, 2.2.4] }
+       - { package-name: StackExchange.Redis.StrongName, versions: [1.0.488, 1.1.608, 1.2.6] }
```

Comment:
I guess I would expect these versions sync up with, or at least include the versions that we are testing in our integration tests.

Reply:
You caught me. I didn't notice the discrepancy until after I pushed. :)